server/app.js

```diff
@@ -3,10 +3,11 @@ import express, { json, urlencoded } from "express";
 import { join } from "path";
 import cookieParser from "cookie-parser";
 import logger from "morgan";
-import dbConnection from "./db/mongoose";
 
+import dbConnection from "./db/mongoose";
 import indexRouter from "./routes/index";
 import pingRouter from "./routes/ping";
+import register from "./routes/register"
```

Comment:
missing semi-colon, what's your thoughts on using es-lint?

Reply:
Oh, I'll turn it on